dev/keys/keys.c

```diff
@@ -63,5 +67,9 @@ int keys_get_state(uint16_t code)
 		dprintf(INFO, "Invalid keycode requested: %d\n", code);
 		return -1;
 	}
+#if WITH_LK2ND_DEVICE
+	return lk2nd_keys_pressed(code);
```

Comment:
avoiding this and doing `keys_init()` followed by `keys_post_state()` would be better for consistency I think (as discussed in chat)